File: CursoPOOUber/Java/Driver.java
package CursoPOOUber.Java;

class Driver extends Account{
    public Driver(String name, String document){
        super(name, document);
    }

    void printDataDriver(){
        System.out.println("User Name: " + name + " User Document: " + document + " User Email: " + email + " User Password: " + password);
    }
}

File: CursoPOOUber/JS/Driver.js
//Definition of the child class "Driver", which inherits from the class "Car".

class Driver extends Account {
    constructor(name, document){
        super(name, document)
        this.driver.name = name;

    }

    printDataDriver() {
        console.log(this.name)
        console.log(this.document)
        console.log(this.email)
        console.log(this.password)
    }
}